internal/tokenizer/failfast_test.go

```
@@ -0,0 +1,147 @@
+}
+
+func copyFile(t *testing.T, src, dst string) {
+	t.Helper()
+	if err := os.MkdirAll(filepath.Dir(dst), 0o755); err != nil {
+		t.Fatalf("mkdir %s: %v", filepath.Dir(dst), err)
+	}
+	in, err := os.Open(src)
+	if err != nil {
+		t.Fatalf("open %s: %v", src, err)
+	}
+	defer in.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `in.Close` is not checked

(errcheck)